File: sdk/java-sdk/src/main/java/com/evernym/verity/sdk/utils/AsJsonObject.java
package com.evernym.verity.sdk.utils;

import org.json.JSONObject;

/**
 * Interface for object that can be converted to a JSON array
 */
public interface AsJsonObject {
    /**
     * Convert this object to a JSON object
     *
     * @return this object as a JSON object
     */
    JSONObject toJson();
}

File: sdk/dotnet-sdk/src/VeritySDK/Utils/AsJsonObject.cs
using System.Json;

namespace VeritySDK
{
    /// <summary>
    /// Interface for save object to JSON
    /// </summary>
    public interface AsJsonObject
    {
        /// <summary>
        /// Convert this object to a JSON object
        /// </summary>
        /// <returns>this object as a JSON object</returns>
        JsonObject toJson();
    }
}
